src/bare-metal/aps/inline-assembly.md

```diff
@@ -22,6 +22,10 @@ for all these functions.)
   contents of the registers.
 - This `main` function needs to be `#[unsafe(no_mangle)]` and `extern "C"`
   because it is called from our entry point in `entry.S`.
+  - Just `#[no_mangle]` would be sufficient but
+    [RFC3325](https://rust-lang.github.io/rfcs/3325-unsafe-attributes.html) uses
+    this notation to draw reviewer attention to attributes which might cause
+    undesired behavior if used incorrectly.
```

Comment:
Not just undesired behaviour, but undefined behaviour. The `unsafe(no_mangle)` notation will become mandatory in a future edition of Rust.

Reply:
Oh, definitely used the wrong word there -- nice catch!

Reply:
https://github.com/google/comprehensive-rust/pull/2566